react_native_fundamentals/index.md

```diff
@@ -1,7 +1,22 @@
 # Module content
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+1. Introduction 
```

Comment:
How about adding a comparison between expo and bare react native? It's a very common question for beginners